### PR TITLE
Fix chisels n bits compatibility (#407)

### PR DESCRIPTION
**Pull request by supsm**
The game no longer crashes on startup when chisels n bits is loaded. Everything seems to work as normal from my rudimentary testing, but crouching causes "bits" to be enlarged to a full block (probably a separate issue).  
Sorry grondag.

### CODE REVIEW

File: src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java [hunk 1]
```diff
@@ -21,31 +21,28 @@
...
 import net.minecraft.client.renderer.block.ModelBlockRenderer;
 import net.minecraft.client.resources.model.BakedModel;
 import net.minecraft.core.BlockPos;
 import net.minecraft.util.RandomSource;
 import net.minecraft.world.level.BlockAndTintGetter;
 import net.minecraft.world.level.block.state.BlockState;
 
-import grondag.canvas.apiimpl.rendercontext.CanvasBlockRenderContext;
-
-// WIP: change this an inject with cancel so that mods that hook in here don't break directly - esp C&B
```
Comment (spiralhalo) [inline src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java:38]: Wow it's pre-approved by Grondag by about a year

File: src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java [hunk 2]
```diff
@@ -21,31 +21,28 @@
...
 // WIP: move hooks upstream so that multi-consumer is available in more cases
 @Mixin(ModelBlockRenderer.class)
 public abstract class MixinModelBlockRenderer {
-	/**
-	 * @author grondag
-	 * @reason performance; less bad than inject and cancel at head
```
Comment (spiralhalo) [inline src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java:44]: Or not??

**Review by spiralhalo — APPROVED** [review]

File: src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java [hunk 3]
```diff
@@ -21,31 +21,28 @@
...
 
+import grondag.canvas.apiimpl.rendercontext.CanvasBlockRenderContext;
 import net.minecraft.client.renderer.block.ModelBlockRenderer;
 import net.minecraft.client.resources.model.BakedModel;
 import net.minecraft.core.BlockPos;
 import net.minecraft.util.RandomSource;
 import net.minecraft.world.level.BlockAndTintGetter;
 import net.minecraft.world.level.block.state.BlockState;
 
-import grondag.canvas.apiimpl.rendercontext.CanvasBlockRenderContext;
```
Comment (spiralhalo) [inline src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java]: Import shouldn't be moved to respect style rules

**Review by spiralhalo — CHANGES_REQUESTED** [review]

File: src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java (hunk 2 above)
Comment (supsm) [reply src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java:44]: I think he didn't *want to*, but kinda had to for mod support

File: src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java (hunk 3 above)
Comment (supsm) [reply src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java]: apologies, my editor was being a bit funny and moved it for me. Should be fixed now.

**Review by spiralhalo — APPROVED** [review]

**Pull request merged**